backend/migrating/v2/migrate-to-v2.sh

```diff
@@ -37,11 +34,16 @@ echo -e "${blue_text}Configuring environment variables for the migration${defaul
 cd $manage_script_dir
 activate_venv
 
+# TODO: Update to support running from container instead of setting these envs
+# VERSION=test docker compose -f docker-compose.yaml run backend .venv/bin/python manage.py create_schema
 echo -e "${blue_text}Running schema creation command...${default_text}"
-python manage.py create_v2_schema || { echo "Schema creation failed"; exit 1; }
+DB_HOST="localhost" REDIS_HOST="localhost" CELERY_BROKER_URL="redis://localhost:6379" \
+python manage.py create_schema || { echo "Schema creation failed"; exit 1; }
```

Comment:
@chandrasekharan-zipstack Can replace all of these direct invocations with `docker compose run` commands as that is the default for end users?

Reply:
@hari-kuriakose I considered that too - it should be ideal. However I was challenged with ways to determine the version that had to be used and realized that we'd duplicate a lot of code from `run-platform.sh`, hope that's okay